### PR TITLE
Some tweaks to baselines/tip 2. Get rid of informal language in first sentence

### PR DESCRIPTION
**Pull request by pstew**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/04.baselines.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## Tip 2: Use traditional methods to establish performance baselines {#baselines}
 
-Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline.
-For example, researchers can build multinomial logistic regression or random forest models using the same software framework that is being used for DL and evaluate its classification performance.
+It is important to implement a simple model to establish an adequate performance baseline, since it is easy to fall into the trap of an unnecessarily convoluted analysis.
+For example, a researcher could first a build a multinomial logistic regression or random forest model using the same software framework planned for DL to first evaluate the classification performance.
```
Comment (agitter) [inline content/04.baselines.md]: This isn't your edit, but do popular DL frameworks support random forest?  That recommendation seemed out of place to me in the initial draft.  Logistic regression makes sense.

Comment (Benjamin-Lee) [reply content/04.baselines.md]: Not that I know of. I didn't write this but I was thinking of the Scikit-Learn API for fitting simple models, [AutoML models](https://epistasislab.github.io/tpot/), and DL models (via Keras)

Comment (pstew) [reply content/04.baselines.md]: Not to my knowledge.

Comment (rasbt) [reply content/04.baselines.md]: Also, RF is probably not feasible on most DL datasets (i.e., thinking of image classification with >500x500 pixels)

Comment (agitter) [reply content/04.baselines.md]: I propose changing to `a researcher could first a build a logistic regression model using the same software framework`.  The `multinomial` part will depend on the data type.

Comment (rasbt) [reply content/04.baselines.md]: sounds good. I don't think we need to say "multinomial" explicitly.

**Pull request merged**